File: app/src/main/java/com/example/mutidemo/ui/AirDashBoardActivity.java
package com.example.mutidemo.ui;

import com.example.mutidemo.base.AndroidxBaseActivity;
import com.example.mutidemo.databinding.ActivityAirDashBinding;
import com.example.mutidemo.util.ColorUtil;

public class AirDashBoardActivity extends AndroidxBaseActivity<ActivityAirDashBinding> {

    @Override
    public void initData() {
        viewBinding.dashBoardView.setMinValue(0);
        viewBinding.dashBoardView.setMaxValue(500);
        int aqiValue = 128;
        viewBinding.dashBoardView.setCurrentValue(aqiValue);
        viewBinding.dashBoardView.setCenterText("良");
        viewBinding.dashBoardView.setAirRingForeground(ColorUtil.aqiToColor(this, aqiValue));
        viewBinding.dashBoardView.setAirCenterTextColor(ColorUtil.aqiToColor(this, aqiValue));
        viewBinding.dashBoardView.setAirCurrentValueColor(ColorUtil.aqiToColor(this, aqiValue));
    }

    @Override
    public void initEvent() {

    }
}

File: app/src/main/java/com/example/mutidemo/view/AirDashBoardActivity.kt
package com.example.mutidemo.view

import android.os.Bundle
import com.example.mutidemo.R
import com.example.mutidemo.util.ColorUtil
import com.pengxh.kt.lite.base.KotlinBaseActivity
import kotlinx.android.synthetic.main.activity_air_dash.*

class AirDashBoardActivity : KotlinBaseActivity() {

    override fun initData(savedInstanceState: Bundle?) {
        dashBoardView.setMinValue(0)
        dashBoardView.setMaxValue(500)
        val aqiValue = 128
        dashBoardView.setCurrentValue(aqiValue)
        dashBoardView.setCenterText("良")
        dashBoardView.setAirRingForeground(ColorUtil.aqiToColor(this, aqiValue))
        dashBoardView.setAirCenterTextColor(ColorUtil.aqiToColor(this, aqiValue))
        dashBoardView.setAirCurrentValueColor(ColorUtil.aqiToColor(this, aqiValue))
    }

    override fun initEvent() {

    }

    override fun initLayoutView(): Int = R.layout.activity_air_dash

    override fun observeRequestState() {

    }

    override fun setupTopBarLayout() {

    }
}